### ISSUE TITLE
2 Google Accounts

### ISSUE THREAD

**Issue by jorrflv** [luka1199/geo-heatmap#29]
Is there anyway to have it process 2 .json files.  or could someone help me with the structure so I can combine the .json files from 2 separate google accounts?

**Comment by luka1199** [comment]
Run the command like this:
```bash
python geo_heatmap.py <file1> <file2>
```
Example:
```bash
python geo_heatmap.py locations1.json locations2.json
```

**Comment by jorrflv** [comment]
Perfect, thank you, I did not know you could just load 2 data sets like that in the command line.

**Comment by luka1199** [comment]
I implemented it a couple of days ago (#12).
If you run ```python geo_heatmap.py -h``` you can see all command line argument options.
